backend/src/services/worker/worker.service.ts

```diff
@@ -184,6 +214,35 @@ class Worker {
       throw new EAMError();
     }
 
+    const master = await this.#masterService.getMasterById(id);
+
+    if (master) {
+      throw new EAMError({
+        status: HttpCode.DENIED,
+        message: ExceptionMessage.MASTER_DELETE,
+      });
+    }
```

Comment:
Why do we need this check?
Maybe add check for if this worker already deleted?

Reply:
Master can not be deleted, this DELETE request is only for workers
![image](https://user-images.githubusercontent.com/83449323/159041146-e1e09636-8c98-4220-a940-107ce0dfabe3.png)